src/bp/ui-studio/src/web/components/Layout/StatusBar/ActionItem.jsx

```diff
@@ -21,7 +21,11 @@ export default props => (
     }
   >
     <div
-      className={classNames(style.clickable, style.item, props.className)}
+      className={classNames(
+        { [style.clickable]: !props.notclickable || props.notclickable === 'false' },
```

Comment:
same thing here having a negation in your variable naming makes it less intuitive for the reader.